addons/postgresql/scripts/etcd-clean.sh

```diff
@@ -1,34 +1,50 @@
 #!/bin/bash
-
 if [ -z "$ETCD_SERVER" ]; then
   exit 0
 fi
 
-IFS=',' read -ra ADDR <<< "$KB_CLUSTER_COMPONENT_POD_IP_LIST"
-FIRST_IP=${ADDR[0]}
+echo "Find command kubectl"
+if command -v kubectl &> /dev/null; then
+  IFS=',' read -ra POD_NAMES <<< "$KB_CLUSTER_POD_NAME_LIST"
 
-PATRONI_API_URL="http://$FIRST_IP:8008/patroni"
-PATRONI_SHUTDOWN_URL="http://$FIRST_IP:8008/shutdown"
+  for POD_NAME in "${POD_NAMES[@]}"; do
+    echo "Attempting to execute pkill -f /usr/local/bin/patroni in pod: $POD_NAME"
 
-status_code=$(curl -s -o /dev/null -w "%{http_code}" $PATRONI_API_URL)
+    MAX_ATTEMPTS=3
+    ATTEMPT=1
+    SUCCESS=0
+    
+    while [ "$ATTEMPT" -le "$MAX_ATTEMPTS" ]; do
+      echo "Attempt $ATTEMPT: Executing pkill -f /usr/local/bin/patroni in pod: $POD_NAME"
+      kubectl exec $POD_NAME -- pkill -f /usr/local/bin/patroni
```

Comment:
Do not use kubectl exec in etcd-clean.sh.
why not shutdown all patroni processes one by one using PATRONI_SHUTDOWN_URL?